spacy/displacy/__init__.py

```diff
@@ -184,8 +184,17 @@ def parse_ents(doc: Doc, options: Dict[str, Any] = {}) -> Dict[str, Any]:
     doc (Doc): Document do parse.
     RETURNS (dict): Generated entities keyed by text (original text) and ents.
     """
+    kb_url_format_template = options.get("kb_url_format_template", None)
```

Comment:
Maybe just `kb_url_template`?

Reply:
Also, we need to document this over at https://spacy.io/api/top-level#displacy_options-ent, right?